src/Sarif.FunctionalTests/ConverterTestData/PREfast/Standardc6306.cxx.xml.sarif

```diff
@@ -29,6 +29,26 @@
       },
       "results": [
         {
+          "ruleId": "6273",
```

Comment:
Why does the new converter produce a result that the old one didn't?

Reply:
That's a good question.  When inspecting the original PREfast xml file, it does contain two defects.  It seems like this result is supposed to be there.  I don't see anything that would cause the C++ project to miss outputting this.

---
In reply to: [146913257](https://github.com/Microsoft/sarif-sdk/pull/657#discussion_r146913257) [](ancestors = 146913257)